src/service/viewer-impl.js

```diff
@@ -682,6 +682,12 @@ export class Viewer {
    * @private
    */
   isTrustedViewerOrigin_(urlString) {
+    // TEMPORARY HACK due to a misbehaving native app. See b/32626673
+    // In native apps all security bets are off anyway, and in browser
+    // origins never take the form that is matched here.
+    if (this.isWebviewEmbedded_ && /^www\.[.a-z]+$/.test(urlString)) {
```

Comment:
The regex ensures that this origin came from a fake source (like a webview) as opposed to a real browser.